### DISCUSSION TITLE
`address.nearbyGPSCoordinate()` return type

### DISCUSSION THREAD

**Discussion by xDivisionByZerox** [faker-js/faker#1058] [General]
In [todo list v2](https://github.com/faker-js/faker/issues/1044) there is a todo that aims to change the input parameter type of the `coordinate` property to `[string, string]` for the `address.nearbyGPSCoordinate()` function . The idea was that the function can provide an input parameter for itself since its return type is `[string, string]`.

My question is: Why doesn't the function return a number tuple in the first place? This seems much more reasonable that changing the input type (which would still be an option).

**Comment by Shinigami92** [comment]
@xDivisionByZerox yes, I'm with you, it was maybe because the output is mostly used to be displayed somewhere.
But even then, `toFixed(4)` can be called on the resulting numbers :shrug: 
It still would be a breaking change, we could do that in v8 and note it in release.
